index.php

```diff
@@ -2,19 +2,66 @@
     //Enable error reporting
     ini_set('error_reporting', E_ALL);
     
-    if ( isset($_GET['message']) && $_GET['message'] == 'update' ) {
-
-        $message = 'You have successfully updated your details';
-    }else if ( isset($_GET['message']) && $_GET['message'] == 'register' ) {
+    if ( isset($_GET['message']) && $_GET['message'] == 'register' ) {
 
         $message = 'Successfully Registered';
-    }else if ( isset($_GET['message']) && $_GET['message'] == 1 ) {
+    } else if ( isset($_GET['message']) && $_GET['message'] == 1 ) {
+
+        $message = 'It seems that you are trying to upload a very large file,<br>
+            . Please upload an image of size less than 2 MB ';
+    } else if ( isset($_GET['message']) && $_GET['message'] == 2 ) {
 
-        $message = 'Your image size exceeds the limit, please upload an image which is less than 2 MB';
-    }else {
+        $message = 'Please login to access your account';
+    } else {
 
         $message = 'Welcome to employee registration portal';
     }
+
+    //Setup Navigation links
+    $navLink1 = 'registration_form.php';
+    $navLink1Name = 'SIGN UP';
+
+    $navLink2 = 'login.php';
+    $navLink2Name = 'LOG IN';
+
+    //Start the session
+    session_start();
+    if ( isset($_SESSION['employeeId']) ) {
+
+        //Create a greeting message with respect to time of the day
+        $time = date("H");
+        if( $time < 12 ) {
+
+            $wish = 'Good morning';
+        } else if ( $time >= 12 && $time < 16) {
+
+            $wish = 'Good afternoon';
+        } else if ( $time >= 16 ) {
+
+            $wish = 'Good evening';
+        }
+
+        if ( isset($_GET['message']) && $_GET['message'] == 'update' ) {
+            $wish = 'You have successfully updated your details';
+        }
+        if ( isset($_GET['message']) && $_GET['message'] == '1' ) {
+            $wish = 'It seems that you are to upload a very large file <br>
+                Please upload an image of size less than 2 MB';
+        }
+        
+        if ( isset($_GET['message']) && $_GET['message'] == '2' ) {
+            $wish = 'You can only access your account';
+        }
+
+        $message = 'Welcome ' . $_SESSION['employeeFirstName'] . ' , ' . $wish;
+
+        //Change Navigation links
+        $navLink1 = 'details.php';
+        $navLink1Name = 'DETAILS';
+
+        $navLink2 = 'logout.php';
```

Comment:
Can create an array for navbar menu items so that it becomes easy to add/remove any menu items.